### PR TITLE
communications guidelines for release process

### PR DESCRIPTION
**Pull request by clrblmt**
Updating promotion process with comms (ref issue #4812)

### CODE REVIEW

File: packages/docs/community/release-process/smart-contracts.md [hunk 1]
```diff
@@ -137,13 +142,29 @@ Deploying a new contract release should occur with the following process:
...
       </ol>
     </td>
   </tr>
 </table>
 
 If the contents of the release (i.e. source Git commit) change at any point after the release has been tagged in Git, the process should increment the release identifier, and process should start again from the beginning. If the changes are small or do not introduce new code (e.g. reverting a contract to a previous version) the audit step may be accelerated.
 
+### Communication guidelines
+
+Communicating your upcoming governance proposal to the community is critical and may help getting it approved.
```
Comment (nambrot) [inline packages/docs/community/release-process/smart-contracts.md]: I probably wouldn't use `you` in this context

**Review by nambrot — APPROVED** [review]
Just a minor nit!

File: packages/docs/community/release-process/smart-contracts.md [hunk 2]
```diff
@@ -101,7 +101,8 @@ Deploying a new contract release should occur with the following process:
         <li>Receive report from auditors.</li>
         <li>If all issues in the audit report have straightforward fixes, announce a forthcoming smart contract release. </li>
         <li>Commit audit fixes to <code>master</code> and cherry-pick to the release branch.</li>
-        <li>Submit audit fixes to auditors for review. </li>
+        <li>Submit audit fixes to auditors for review.</li>
+        <li>Let the community know about the upcoming release proposal by posting details to the Governance category on https://forum.celo.org and cross post in the Discord <code>#governance</code> channel. See the 'Communication guidelines' section below for information on what your post should contain.</li>
```
Comment (YazzyYaz) [inline packages/docs/community/release-process/smart-contracts.md]: maybe add link to Discord channel here.

File: packages/docs/community/release-process/smart-contracts.md [hunk 3]
```diff
@@ -128,7 +131,9 @@ Deploying a new contract release should occur with the following process:
     <td>
       <ol>
         <li>Confirm all contracts working as intended on Alfajores.</li>
+        <li>Confirm audit is complete and make sure your forum post contains a link to it.</li>
```
Comment (YazzyYaz) [inline packages/docs/community/release-process/smart-contracts.md:134]: We should probably provide guidelines on security audits to perform and which auditors have worked well in the past. This isn't needed for now but probably for a future PR.

A lot of new dapps show they're audited and use it as a marketing scheme as a badge of honor without providing a lot of information on the auditors. Other times, they ask for an audit a few days before going live, not leaving enough time for professionals to properly audit the smart contract changes.

File: packages/docs/community/release-process/smart-contracts.md [hunk 4]
```diff
@@ -137,13 +142,29 @@ Deploying a new contract release should occur with the following process:
...
+Each smart contract release governance proposal should be accompanied by a [Governance category](https://forum.celo.org/c/governance/) forum post that contains the following information:
+* Name of proposer (individual contributor or organization).
+* Background information.
+* Link to the release notes.
+* Link to the audit report(s).
+* Anticipated timings for the Baklava and Alfajores testnets and Mainnet.
+
+It's recommended to post as early as possible and at minimum one week before the anticipated Baklava testnet governance proposal date. 
+
+Make sure to keep the post up to date. All updates (excluding fixing typos) should be communicated to the community in the [Discord](http://chat.celo.org/) `#governance` channel.
```
Comment (YazzyYaz) [inline packages/docs/community/release-process/smart-contracts.md:166]: Maybe there needs to be a way to request having the new governance proposal as part of the Celo mailing list for validators and stakeholders so we can add more spotlight on changes. I'd be careful though about not using the mailing list though to show that we are endorsing any of the proposals, more to show that they exist for the sake of validators and node operators.

**Review by YazzyYaz — APPROVED** [review]

**Comment by YazzyYaz** [comment]
LGTM. Minor nitpicks and observations but overall pretty good.

File: packages/docs/community/release-process/smart-contracts.md (hunk 2 above)
Comment (clrblmt) [reply packages/docs/community/release-process/smart-contracts.md]: I tried but can't add links in an md table

File: packages/docs/community/release-process/smart-contracts.md (hunk 3 above)
Comment (clrblmt) [reply packages/docs/community/release-process/smart-contracts.md:134]: Good point. Noted.

File: packages/docs/community/release-process/smart-contracts.md (hunk 4 above)
Comment (clrblmt) [reply packages/docs/community/release-process/smart-contracts.md:166]: As far as I know there is no Celo Foundation mailing list. All conversations happen on the forum and Discord. My sense is that all community comms should happen there. cLabs however is starting a Engineering/Product newsletter in which some of this information will be communicated.

**Pull request merged**